react-frontend/src/components/Home/cardList.js

```diff
@@ -76,11 +79,7 @@ const CardList = () => {
             />
           </Col>
         </Row>
-        <Row style={{ paddingTop: '30px' }}>
-          <Col sm={12}>
-            <Title2>Learn from Others</Title2>
-          </Col>
-        </Row>
+
```

Comment:
Is there a reason this is getting deleted?  I believe it was added in to provide a clear separation in the two types of content in the page.  (I don't think the designs were updated to reflect this)

Reply:
I made an executive decision that it was redundant to have a card titled "Learning" under the heading "Learn from Others" -- but you are right in pointing out that this change was not specifically called for. 

@singhinderdeep -- can you please have a look at [the deployment](https://digital-gov-frontend-pr-855-c0cce6-dev.apps.silver.devops.gov.bc.ca) and see if you think this is satisfactory, or if you'd like to reinstate the "Learn from Others" heading with alternate wording, perhaps?

Reply:
Discussed this with Mark and happy with Mark's decision of removing "Learn from others" heading